src/Widgets/FormSelectWidget/FormSelectWidgetClass.ts

```diff
@@ -7,14 +7,23 @@ export const FormSelectWidget = Scrivito.provideWidgetClass(
       selectionType: [
         "enum",
         {
-          values: ["radio", "dropdown", "multi"],
+          values: ["radio", "dropdown", "multi", "linear-scale"],
         },
       ],
       title: "string",
       items: "stringlist",
       customFieldName: "string",
       required: "boolean",
       helpText: "html",
+      linearScaleLowerLimit: ["enum", { values: ["0", "1"] }],
```

Comment:
Is `linear` scale something new or is it based on the existing component? 

Just something to think about. What if someone wants to have a scale to 100? Or 1000?

I think having upper limit, lower limit and options to provide number of steps could help

Reply:
It's based on the Google Forms linear-scale questions [here](https://docs.google.com/forms/d/e/1FAIpQLSdLQORP-lBrFDKk39NMmf5JMUtq1SJ9Kct_dK6ARCVbfh2kNg/viewform)
The maximum is 10. 
It is usually used as some kind of rating, for large scaling numbers a Slider Widget should be implemented similar to the one in the custom widgets example in the Readme , otherwise if we had 100 for example, it would take half the page of space having 100 radio buttons :)